docs/docs.go

```diff
@@ -1368,7 +1368,7 @@ const docTemplate = `{
                         "type": "string",
                         "description": "Filter by miner",
                         "name": "miner",
-                        "in": "query"
+                        "in": "path"
```

Comment:
this is automatic. all the swagger files (docs/*) are auto-generated after the swagger workflow job step.

Reply:
I didn't change that file at all. Maybe it was a pre-commit hook or something?

Reply:
It's a workflow on github: https://github.com/application-research/estuary/blob/a7420a2c22262bd6dcc1edb203eba6bb0f763afe/.github/workflows/swagger.yml#L18-L22